designs/CentrallyManagingNuGetPackageVersions-VSManagePackages.md

```diff
@@ -0,0 +1,108 @@
+# Centrally managing NuGet package versions - Managing packages in Visual Studio Spec
+
+* Status: **In Review**
+* Author(s): [Andy Zivkovic](https://github.com/zivkan)
+
+## Related Specs
+
+* [Centrally managing NuGet package versions](https://github.com/NuGet/Home/wiki/Centrally-managing-NuGet-package-versions)
+* [Centrally managing NuGet package versions - Restore Design Spec](https://github.com/NuGet/Home/blob/dev/designs/CentrallyManagingNuGetPackageVersions-Restore.md)
+
+## Problem Background
+
+To date, NuGet does not modify any MSBuild files (projects files, or files imported by the projects) either directly or though MSBuild APIs. Instead, NuGet uses APIs provided by the various project systems when projects need to be modified as a result of installing, updating or removing packages. In practise, NuGet has only indirectly caused project files to be modified.
+
+With centrally managed package versions, there will be a single file, `Directory.Packages.props`, which is intended to be shared across multiple projects. There are multiple design decisions about which Visual Studio component will have the responsibility for writing changes to the file, and how the updated values will flow throughout Visual Studio. 
+
+## Who are the customers
+
+All customers managing packages on projects that are using central package version management with projects in Visual Studio IDE.
+
+## Goals
+
+1. When installing a package in a centrally managed project, if the package already exists in the `Directory.Packages.props` file, only the project file is modified. The `PackageReference` element added must not have a version applied.
+2. When installing a package in a centrally managed project, if the package does not already exist in the `Directory.Packages.props` file, then the package version is added to `Directory.Packages.props`, in addition to goal 1 above.
+3. When updating a centrally managed package version, the `Directory.Packages.props` file is updated, and all projects that use the package are restored with the new package version.
```

Comment:
I think we should call it out explicitly, but 100% of the projects affected by the Directory.Packages.Props files should be restored because CPVM affects transitive packages as well.